cmd/traffic/cmd/manager/internal/config/trafficManager.go

```diff
@@ -0,0 +1,49 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/telepresenceio/telepresence/rpc/v2/manager"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli"
```

Comment:
The traffic manager should never use anything beneath the `pkg/client` package. Common functions must be placed in a relevant package directly under `pkg`.